configs/eval_pre.yaml

```diff
@@ -53,7 +53,7 @@ utility:
 
   asr:
     dataset_name: [libri_dev, libri_test, IEMOCAP_dev, IEMOCAP_test]
-    model_name: asr_pre_ctc_wav2vec2 # name for ASR model, check 'exp/' dir
+    model_name: asr_pre_ctc_wav2vec2 # name for ASR model, asr_pre_transformer_transformerlm + EncoderDecoderASR or asr_pre_ctc_wav2vec2 + EncoderASR
 
     evaluation:
       model_type: EncoderASR # EncoderDecoderASR or EncoderASR (from speechbrain)
```

Comment:
The `hparams_file` option can be removed from the eval_pre.yaml. (but not the .py file ;))

Reply:
I am not sure whether I understand this request, this is just comment and I think keep this here is fine.